rosdep/python.yaml

```diff
@@ -1449,6 +1449,10 @@ python-scipy:
     utopic_python3: [python3-scipy]
     vivid: [python-scipy]
     vivid_python3: [python3-scipy]
+python-selenium:
+  ubuntu:
+    pip:
+      packages: [selenium]
```

Comment:
Well the obvious thing is the version in Trusty is 2.25.0 and 2.46.0 is the latest release. That'd be my guess as to why they want the pip one. But I agree that if there is a Trusty deb and it works we should have that as the default key. Unfortunately there is no good practice for when you need a newer version and pip can provide it, but there is also an apt-get version others might use.